### ISSUE TITLE
Train images other than 640x640

### ISSUE THREAD

**Issue by ManuBN786** [lyuwenyu/RT-DETR#635]
I am resizing my images to 1920 by making changes in dataloader.yml 

{type: Resize, size: [1920, 1920] }
scales: [1440, 1536, 1632, 1728, 1824, 1920, 1920, 1920, 2016, 2112, 2208, 2304, 2400]

and then in rtdetrv2_r50vd.yml as
eval_spatial_size: [1920, 1920]

but even after several epochs I still get mAP as zero

IoU metric: bbox
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = -1.000
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = -1.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=  1 ] = 0.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 10 ] = 0.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = -1.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = -1.000
 Average Recall     (AR) @[ IoU=0.50      | area=   all | maxDets=100 ] = 0.000
 Average Recall     (AR) @[ IoU=0.75      | area=   all | maxDets=100 ] = 0.000

how to fix this please?

**Comment by wenhwu** [comment]
Same issue here, did you get it fixed? thanks

**Comment by Hlnvor** [comment]
Hi, I have the same issue here. Has anyone found a solution for high-resolution images (other than splitting the image into patches)?

**Comment by ManuBN786** [comment]
I could not resolve the issue.

So I am using RF-Detr now.
